### DISCUSSION TITLE
[求助]学业困难

### DISCUSSION THREAD

**Discussion by FallingLine** [share121/inter-knot#3247] [崩坏：星穹铁道]
![Uploading IMG_20240820_083545.jpg…]()
![Uploading IMG_20240820_083558.jpg…]()
求助！这两道题该怎么写？？数学物理方法有巨大的困难！

**Comment by FallingLine** [comment]
奇怪，为什么图片发送不出去？

**Reply by XYdtman** [reply]
点预览，等图片刷出来了再发

**Reply by XYdtman** [reply]
还有一种可能，就算图片太大了，好像一张不能超过10mb

**Comment by Nie7bai** [comment]
图片发送的话要等Uploading消失以后再发送
